gym/envs/classic_control/pendulum.py

```diff
@@ -6,6 +6,28 @@
 
 
 class PendulumEnv(gym.Env):
+    """
+    Description:
+        The inverted pendulum swingup problem is a classic
+        problem in the control literature. In this version of the
+        problem, the pendulum starts in a random position, and the
+        goal is to swing it up so it stays upright.
+    Source:
+    Observation:
+        Type: Box(3)
+        Num     Observation               Min                     Max
+        0                                 -1                       1
+        1                                 -1                       1
```

Comment:
The first two observations are missing, what is their interpretation?

Reply:
Those are the Mass and the Length of the Pendulum.